packages/itwinui-react/src/core/Buttons/IdeasButton.tsx

```diff
@@ -29,7 +29,8 @@ export const IdeasButton = React.forwardRef((props, ref) => {
   return (
     <Button
       ref={ref}
-      data-iui-variant='idea'
+      className='iui-button-idea'
```

Comment:
`className` needs to be properly merged here

Reply:
I think I did this correctly in https://github.com/iTwin/iTwinUI/pull/1919/commits/ad7e0ae9af16b5cc045b62eea13f2de0ca9ec73d but double check for me please.